log/format.go

```diff
@@ -375,8 +167,11 @@ func formatLogfmtValue(value interface{}, term bool) string {
 			return escapeString(s.TerminalString())
 		}
 	}
-	value = formatShared(value)
 	switch v := value.(type) {
+	case error:
+		return v.Error()
+	case fmt.Stringer:
+		return v.String()
```

Comment:
Wait. Both of these needs `escapedString`, right? We should add testcase (on `master`) which logs:

- an error with spaces `errors.New("error: an 'error' occurred")` and 
- a `fmt.Stringer` which also spits out text spaces (needs quoting)